### PR TITLE
Fixed : Registeration & Login on the Hosted App

### LINKED ISSUE #29: Bug: Regiteration & Login Does not work on the hosted App
### Describe the bug

A user is unable to login on the hosted website as the requests are directed to wrong API (https://anitab-org.github.io/users/login). I have registered a test user on Backend hosted on Heroku & verified the user & when I try to login with that user it shows error like 'please check your password & username '.

### Expected behavior

All the requests coming from the Frontend should be directed to Backend hosted on Heroku & the login section should show the correct error that is coming from the Backend while registering or while logging in to the application. This type of error message when a user has entered correct credentials ,will effect User experience badly.

### Screenshots

Peek 2021-02-12 21-03
Screenshot from 2021-02-12 21-06-45
Peek 2021-02-12 21-04
### Desktop (please complete the following information):
 - OS: Ubuntu 20.04
 - Browser Firefox

### Additional Context

### PR DESCRIPTION
**Pull request by chinmaym07**
### Description

I have made a .env.sample file & for testing
1) if you are testing it locally you need to set REACT_APP_ENVIRONMENT_CONFIG=local 
2) if you are testing it on the production you need to set REACT_APP_ENVIRONMENT_CONFIG=prod & as CORS issue is not yet solved on the backend [#93](https://github.com/anitab-org/stem-diverse-tv/issues/93) you need to have this [chrome](https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf/related?hl=en) extension & enable then test you will be able to register & login to the hosted version.
3) It requires a documentation update on the readme.md about setting the env config.

Fixes #29 

### Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

I have added gif
1) When REACT_APP_ENVIRONMENT_CONFIG is set to local .

Peek 2021-02-23 15-30

2) When REACT_APP_ENVIRONMENT_CONFIG is set to prod .

Peek 2021-02-23 15-35

### CODE REVIEW

**Review by jalajcodes — CHANGES_REQUESTED** [review]
Can you edit the `package.json` file to remove `proxy` as we won't be needing it now after we add CORS on backend

After above change everything LGTM. 😄

**Comment by chinmaym07** [comment]
@jalajcodes thanks for the review  .
I have made the required changes as well as added the instructions required for setting it up in the Readme.
If you can review it again .It will be really helpful to me . thanks in advance. 👍🏻  😬

**Review by jalajcodes — CHANGES_REQUESTED** [review]
Can you amend your commit and remove the `.lock` files as they are unrelated

**Comment by chinmaym07** [comment]
@jalajcodes Done . Please have a look .
Can you please test the same locally ?

**Review by annabauza — CHANGES_REQUESTED** [review]
yarn.lock  should stay

**Comment by jalajcodes** [comment]
> yarn.lock  should stay

hmm...no problem, I just asked to remove it because there are no new packages installed or removed in this PR

**Review by annabauza — APPROVED** [review]

**Comment by annabauza** [comment]
@chinmaym07  can you resolve conflicts? Thanks.

**Comment by jalajcodes** [comment]
@chinmaym07 Any updates?

**Comment by chinmaym07** [comment]
> @chinmaym07 Any updates?
> 

Yes will resolve it by today .. :+1:

**Comment by chinmaym07** [comment]
I was testing someone else PR & forgot to unstage those unnecessary changes & accidentally made those additional changes & therefore removed those in the second commit. Sorry for the trouble !!

**Comment by annabauza** [comment]
> > yarn.lock  should stay
> 
> hmm...no problem, I just asked to remove it because there are no new packages installed or removed in this PR

In meaning we do not delete that file. But if there are unintentional changes those changes should be removed.

**Comment by annabauza** [comment]
> I was testing someone else PR & forgot to unstage those unnecessary changes & accidentally made those additional changes & therefore removed those in the second commit. Sorry for the trouble !!
> 

Your pr deletes 17k lines of code...?

**Review by annabauza — CHANGES_REQUESTED** [review]
@chinmaym07 can you please address my questions and let's make it ready ASAP. This is a massive blocker. Thank you.

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -105,37 +101,6 @@ const Login = () => {
...
-                                        <IconButton
-                                        aria-label="toggle password visibility"
-                                        onClick={()=>setShowPassword(!showPassword)}
-                                        >
-                                        {showPassword ? <Visibility /> : <VisibilityOff />}
-                                        </IconButton>
-                                    </InputAdornment>        
-                                }
-                            />
-                        </FormControl>
```
[the remark this answers is not in the document]
Comment (chinmaym07) [reply src/pages/Login.tsx:138]: That's a part of another pr .. I was testing #39 . while testing I forgot to unstage the changes in my pr ..
That's why remove this in the second commit,

**Comment by chinmaym07** [comment]
I will open another pr by today itself as I know that's a blocker .. will complete it by today ... 
I am really sorry for the trouble that is caused by me . @annabauza

**Comment by annabauza** [comment]
You helping no trouble. Thank you.

**Pull request closed without merging**